pkg/vpcmodel/output.go

```diff
@@ -146,15 +148,15 @@ func AggregateVPCsOutput(outputList []*SingleAnalysisOutput, f OutFormat, uc Out
 	switch f {
 	case Text, MD, Debug:
 		// plain concatenation
+		infoMessage := ""
 		vpcsOut := make([]string, len(outputList))
 		for i, o := range outputList {
 			vpcsOut[i] = o.Output
+			if uc != SingleSubnet && o.hasStatelessConn {
+				infoMessage = asteriskDetails
```

Comment:
add a function that returns the value of `infoMessage`,  to be used in both places on this file (instead of having similar code in both places).